lib/ruby-mpd/parser.rb

```diff
@@ -24,15 +24,15 @@ def convert_command(command, *params)
             "#{param.begin}:#{param.end + (param.exclude_end? ? 0 : 1)}"
           end
         when MPD::Song
-          %Q["#{param.file}"] # escape filename
+          %Q["#{param.file.gsub('"','\\"')}"] # escape filename
```

Comment:
Yeah, I think extracting it in a utility function would be good, so that we can update it easily in the future, besides that LGTM :+1: